.github/workflows/build-images-workflow-run.yml

```diff
@@ -209,6 +209,18 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: ${{ needs.cancel-workflow-runs.outputs.sourceHeadSha }}
+      - name: "Retrieve DEFAULTS from the _initialization.sh"
+        # We cannot "source" the script here because that would be a security problem (we cannot run
+        # any code that comes from the sources coming from the PR. Therefore we extract the
+        # DEFAULT_BRANCH and DEFAULT_CONSTRAINTS_BRANCH via custom grep/awk/sed commands
+        run: |
+          DEFAULT_BRANCH=$(grep "export DEFAULT_BRANCH" scripts/ci/libraries/_initialization.sh | \
+            awk 'BEGIN{FS="="} {print $3}' | sed s'/["}]//g')
```

Comment:
Don't you mean $2 to match `export DEFAULT_BRANCH="master"`?

Reply:
Not really the form of it is export DEFAULT_BRANCH=${DEFAULT_BRANCH:="master"} so that it can be overridden.

Reply:
Ah, I was looking at a different file (or maybe an older version) then.